src/main.rs

```diff
@@ -460,7 +474,14 @@ fn run() -> Result<(), failure::Error> {
 
         let watch = matches.is_present("watch");
 
-        commands::preview(target, user, method, body, watch)?;
+        commands::preview(
+            target,
+            user,
+            method,
+            body,
+            watch,
+            matches.is_present("verbose"),
```

Comment:
i think it'd be cleaner to take this matches.is_present out of the functional cal and set it up in a variable, a nit, but we are already passing so much i think it would read better